src/shared/components/rightbar/RightBar.tsx

```diff
@@ -157,6 +157,29 @@ export default class RightBar extends React.Component<
         }
     }
 
+    private getInstallationMap() {
+        return AppConfig.serverConfig.skin_right_nav_show_installations &&
```

Comment:
So I think rather than make this a configuration, we're going to show the map only if this is cbioportal.org. To do this, you can just do something like this:

```suggestion
        return AppConfig.serverConfig.app_name === 'public-portal' &&
```

Reply:
Good start. We can talk about this on Monday, mostly after we meet with JJ